app/src/main/java/com/ncnf/authentication/ui/SignInFragment.java

```diff
@@ -83,19 +82,18 @@ private void logIn(String email, String password){
 
         showProgressBar(true);
 
-        CompletableFuture<AuthenticationResponse> futureResponse = auth.logIn(email, password);
+        CompletableFuture<Boolean> futureResponse = auth.logIn(email, password);
 
-        futureResponse.thenAccept(response -> {
-            if(response.isSuccessful()){
-                Log.d(DEBUG_TAG,"Successful login for " + email);
-                Intent intent = new Intent(getActivity(), this.activity);
-                startActivity(intent);
-            } else {
-                Log.d(DEBUG_TAG,"Unsuccessful login for " + email + " : " + response.getException().getMessage());
+        futureResponse.thenAccept(result -> {
+            Log.d(DEBUG_TAG,"Successful login for " + email);
+            Intent intent = new Intent(getActivity(), activity);
```

Comment:
Will we switch to fragments later?

Reply:
This is fragments already, but the layout will change